app/package.json

```diff
@@ -27,14 +27,20 @@
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
     "autoprefixer": "^10.4.2",
-    "cypress": "^13.10.0",
```

Comment:
Unless I'm mistaken and/or there's a reason for this, we shouldn't be removing cypress as a dev dependency should we?

Reply:
Seconding Dakota here. Is there a particular reason we're removing it?

Reply:
Yes it shouldn't be removed. I've been trying to figure out why it removed it